docs-standards/20-style-the-content/11-ui-elements.md

```diff
@@ -0,0 +1,109 @@
+# UI elements
+
+UI elements are screen components the user can interact with. A checkbox, a menu, or a button are all UI elements. An explanatory text with no user interaction is not considered a UI element in the context of this guide.
+
+### General guidelines
+
+- Don't write the UI control's type (button, drop-down menu, radio button, etc.), except when it isn't clear you're referring to the UI control. The exceptions to this rule are *pop-up menu* and *check the [...] box.*
+
+    | Usage       | Example                                                          |
+    |:------------|:-----------------------------------------------------------------|
+    | **Correct** | To publish your token, click **Publish**.                        |
+    | Incorrect   | To publish your token, click the **Publish** button.             |
+    | **Correct** | On the **Permission** pop-up menu, tap the **Write** permission. |
+    | Incorrect   | On the **Permissions** pop-up, tap the **Write** permission.     |
+    | Incorrect   | On **Permissions**, tap the **Write** permission.                |
+
+- When describing actions across different screens or menu options, use angle brackets (>) to describe the sequence of UI elements.
+
+    | Usage       | Example                                                            |
+    |:------------|:-------------------------------------------------------------------|
+    | **Correct** | Go to **Settings > Messages > Privacy**.                           |
+    | Incorrect   | Go to **Settings**, then go to **Messages**, then tap **Privacy**. |
+
+- Don't describe to users what's happening on the screen, unless the result is unexpected.
```

Comment:
The content in line 24-42 is from the Status procedure style guide. Because we don't need to emphasize how to describe UI elements in the IFT procedure doc standards, this section is better suited for this content